Assets/Scenes/MainGameplayScene.unity

```diff
@@ -1310,6 +1310,10 @@ PrefabInstance:
       propertyPath: m_LocalEulerAnglesHint.z
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 4600950328346219042, guid: 3503da22fa60c2b478aaba19e08e2f77, type: 3}
+      propertyPath: m_Sprite
```

Comment:
Can you verify if this is a change that can be propagated to the `BirdMan` prefab instead?